### ISSUE TITLE
Ivy failed with HMR (angular class)

### ISSUE THREAD

**Issue by splincode** [angular/angular#31916]
### Affected Package
`@angularclass/hmr": "^2.1.3`

### Is this a regression?

https://github.com/ngxs/store/blob/master/integration/app/app.component.html

### Description

 A clear and concise description of the problem...

## 🔬 Minimal Reproduction

```
$ git clone https://github.com/ngxs/store.git
$ npm install
$ ng update --next --force
$ npm start
# changes app.component.html
```

## 🔥 Exception or Error
<pre>ERROR Error: The pipe 'async' could not be found!
<code>

</code></pre>

## 🌍  Your Environment

**Angular Version:**
<pre><code>

 Updating package.json with dependency @angular/cli @ "8.2.0-rc.0" (was "8.1.3")...
    Updating package.json with dependency @angular/animations @ "8.2.0-rc.0" (was "8.1.3")...
    Updating package.json with dependency @angular/forms @ "8.2.0-rc.0" (was "8.1.3")...
    Updating package.json with dependency @angular-devkit/build-angular @ "0.802.0-rc.0" (was "0.801.3")...
    Updating package.json with dependency @angular/platform-browser-dynamic @ "8.2.0-rc.0" (was "8.1.3")...
    Updating package.json with dependency @angular/router @ "8.2.0-rc.0" (was "8.1.3")...
    Updating package.json with dependency @angular/core @ "8.2.0-rc.0" (was "8.1.3")...
    Updating package.json with dependency @angular/platform-browser @ "8.2.0-rc.0" (was "8.1.3")...
    Updating package.json with dependency @angular/common @ "8.2.0-rc.0" (was "8.1.3")...
    Updating package.json with dependency @angular/language-service @ "8.2.0-rc.0" (was "8.1.3")...
    Updating package.json with dependency @angular/compiler @ "8.2.0-rc.0" (was "8.1.3")...
    Updating package.json with dependency @angular/platform-server @ "8.2.0-rc.0" (was "8.1.3")...
    Updating package.json with dependency @angular/compiler-cli @ "8.2.0-rc.0" (was "8.1.3")...
    Updating package.json with dependency typescript @ "3.5.3" (was "3.4.5")...

</code></pre>

**Anything else relevant?**

**Comment by kukjevov** [comment]
This might be relative 

https://github.com/angular/angular-cli/issues/15225

Well i think this problems occurs only with Angular 8.1.0 and higher, because for me its working with 8.0.0

**Comment by alxhub** [comment]
Closing as a duplicate of #31654. A fix is on the way (#31932)!
